### DISCUSSION TITLE
Http methods support for http_scrape

### DISCUSSION THREAD

**Discussion by mans2singh** [vectordotdev/vector#14660] [Q&A] [answered]
Hi Folks:

I am trying to use [http_scrape](https://master.vector.dev/docs/reference/configuration/sources/http_scrape/) source which is currently in master branch.   

I want to use http POST so that the http server can return a customized response (metrics/events) based on the body.
However, I did not find any configuration for http methods for http_scrape (like in the [http](https://master.vector.dev/docs/reference/configuration/sources/http) component). It looks like http_scrape only supports GET.

Please let me know if the http_scrape supports other http methods and where can I find some documentation if I have missed it.

Thanks.

**Comment by spencergilbert** [comment, marked as answer]
At this time only GET requests are performed by the source - feel free to open a feature request to add support for additional methods though!
